vulnfeeds/utility/logginghelpers.go

```diff
@@ -13,23 +13,21 @@ import (
 // CreateLoggerWrapper creates and initializes the LoggerWrapper,
 // and also returns a cleanup function to be deferred
 func CreateLoggerWrapper(name string) (LoggerWrapper, func()) {
```

Comment:
Nit: Be consistent with https://pkg.go.dev/cloud.google.com/go/logging#Client.Logger given this is being passed to that.

```suggestion
func CreateLoggerWrapper(logID string) (LoggerWrapper, func()) {
```